hudi-cli/src/main/java/org/apache/hudi/cli/commands/CleansCommand.java

```diff
@@ -129,4 +125,31 @@ public String showCleanPartitions(@CliOption(key = {"clean"}, help = "clean to s
     return HoodiePrintHelper.print(header, new HashMap<>(), sortByField, descending, limit, headerOnly, rows);
 
   }
+
+  @CliCommand(value = "cleans run", help = "run clean")
+  public String runClean(@CliOption(key = "sparkMemory", unspecifiedDefaultValue = "4G",
+      help = "Spark executor memory") final String sparkMemory,
+                         @CliOption(key = "propsFilePath", help = "path to properties file on localfs or dfs with configurations for hoodie client for cleaning",
+                           unspecifiedDefaultValue = "") final String propsFilePath,
+                         @CliOption(key = "hoodieConfigs", help = "Any configuration that can be set in the properties file can be passed here in the form of an array",
```

Comment:
If we are passing the props file, I don't think we need this array as well, we can just read everything from the file

Reply:
@n3nash Please refer to this comment by @bvaradar - https://jira.apache.org/jira/browse/HUDI-118?focusedCommentId=16983739&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16983739. 

I was trying to implement as per what is suggested in this comment.

Reply:
Okay, I think this is to handle the case where props cannot be passed. In that case, can you clean up the checks a little bit, in the way this PR does it : https://github.com/apache/incubator-hudi/pull/691/files#diff-b5f6caf2f1d3556bcec0bec5d75c14f2R104

Reply:
Made the changes. @n3nash